audio-worklet/src/subtractive-synth-demo.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+class SubtractiveSynthDemo {
+  constructor(context) {
+    this.maxCutoff_ = 5000;
```

Comment:
Add a comment on why 5000.  And a comment on what maxCutoff really means.

If this is the maximum cutoff frequency allowed, then 5000 seems rather low since my old ears can still hear up to 12-14 kHz.